core-bundle/contao/templates/twig/content_element/accordion.html.twig

```diff
@@ -0,0 +1,73 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+{% use "@Contao/component/_text.html.twig" %}
+
+{% set attributes = attrs(attributes|default).addClass('content-accordion') %}
+
+{% if not nested_fragments and layout is defined %}
+    {% set layout_attributes = attrs().addClass('content-text media media--' ~ layout, layout and image) %}
+{% endif %}
+
+{% block wrapper_tag %}section{% endblock %}
+
+{% block content %}
+    <div class="{{ toggler }}">{{ title }}</div>
+    <div class="{{ accordion }}"><div{{ layout_attributes|default }}>
+        {% if nested_fragments %}
+            {% for element in nested_fragments %}
+                {{ content_element(element) }}
+            {% endfor %}
+        {% else %}
+            {{ block('text_component') }}
+        {% endif %}
+    </div></div>
+{% endblock %}
+
+{% block script %}
+    {% add "handorgel_js" to body %}
+        <script src="{{ asset('js/handorgel.min.js', 'contao-components/handorgel') }}"></script>
+        <script>
+            (function () {
+                // DO NOT CHANGE THIS FUNCTION!
+                // Set the Handorgel.js options in the function call below.
+                const initAccordion = function (selector, options) {
+                    const headerElements = document.querySelectorAll(selector);
+                    const contentElements = [];
+
+                    if (!headerElements) {
+                        return;
+                    }
+
+                    headerElements.forEach((toggler) => {
+                        const button = document.createElement('button');
+                        button.type = 'button';
+                        button.append(...toggler.childNodes);
+                        toggler.appendChild(button);
+                        toggler.parentNode.classList.add('handorgel');
+                        contentElements.push(toggler.nextElementSibling);
+                    });
+
+                    const accordion = new handorgel(
+                        document.body,
+                        Object.assign({ headerElements, contentElements }, options || {})
+                    );
+
+                    // Show the first element
+                    accordion.folds[0].open();
+                }
+
+                // Change this selector according to your template, or duplicate
+                // the line if you use multiple accordion element classes.
+                initAccordion('.content-accordion > .{{ toggler }}', {
```

Comment:
If this is adjustable, we should probably introduce a block. Wdyt?

Reply:
Do you really mean line 60 or rather line 62 (the custom options)?

Reply:
I'm not sure. The comment suggests that the selector should be adjusted if needed. How can I do this without duplicating the block?

Reply:
But probably also the options, yes. 🙂

Reply:
The comment is a leftover from the `js_accordion` template, which does not apply here, because the toggler class is already customizable in the module settings. But a custom block for the options sure makes sense. 👍

Reply:
Added in ca8c62d9f475e712d94e841f84d7b08603ffab8d.